app/furniture/marketplace/carts/_cart.html.erb

```diff
@@ -16,7 +16,7 @@
         </tr>
       </thead>
       <tbody class="divide-y divide-gray-200 bg-white">
-        <%- cart.marketplace.products.each do |product| %>
+        <%- cart.marketplace.products.kept.each do |product| %>
```

Comment:
Since we already ensure that when a Product is discarded it gets removed of all non-order carts, we shouldn't need to have this extra scope here, right?

When I read this with the scope it makes me think that maybe a Cart (not-order) can contain archived Products, which seems wrong.

Reply:
This is the `marketplace.products`, not the `cart` products; which is a manifestation of the gross-ness of not having a `menu` which is distinct from the `cart`

Reply:
Ah, yes, sorry, I misread!